packages/expo-firebase-polyfill/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "expo-firebase-polyfill",
+  "version": "1.0.0",
+  "description": "Polyfill missing APIs for the Firebase JS SDK",
+  "main": "build/FirebasePolyfill.js",
+  "types": "build/FirebasePolyfill.d.ts",
+  "sideEffects": true,
+  "scripts": {
+    "build": "expo-module build",
+    "clean": "expo-module clean",
+    "lint": "expo-module lint",
+    "test": "expo-module test",
+    "prepare": "expo-module prepare",
+    "prepublishOnly": "expo-module prepublishOnly",
+    "expo-module": "expo-module"
+  },
+  "keywords": [
+    "react-native",
+    "expo",
+    "firebase",
+    "firebase-polyfill",
+    "react-native-firebase-polyfill"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/expo/expo.git",
+    "directory": "packages/expo-firebase-polyfill"
+  },
+  "bugs": {
+    "url": "https://github.com/expo/expo/issues"
+  },
+  "author": "650 Industries, Inc.",
+  "license": "MIT",
+  "homepage": "https://github.com/expo/expo/tree/master/packages/expo-firebase-polyfill/",
+  "devDependencies": {
+    "expo-module-scripts": "^1.2.0"
+  },
+  "dependencies": {
+    "base-64": "^0.1.0"
```

Comment:
I think it'd be better at this point to have a reliable base64 package that all universal modules can rely on.

Reply:
I agree. There seem to be better and more "reliable" packages out there. This looks like a good candidate and supports the `atob` and `btoa` variants:
https://www.npmjs.com/package/js-base64

What exactly do you mean by "that all universal modules can rely on." ? As in just to have reliable`atob` and `btoa` polyfills, or in a broader sense?